taxi/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ("model", "manufacturer")
+    search_fields = ("model",)
+    list_filter = ("manufacturer__name",)
+
+
+admin.site.register(Manufacturer)
```

Comment:
You should use one approach for registering your models, either with a decorator or with a functional approach.